lib/Dialog/Dialog.js

```diff
@@ -201,6 +219,7 @@ module.exports = kind(
 	* @private
 	*/
 	messageChanged: function () {
-		this.$.message.setContent(this.message);
+		if(!this.bodyComponents)
```

Comment:
Please adjust the `if` statement to have a space before the opening parenthesis and, in this case, either enclosing the body with `{}` or moving the body up to the same line as the `if` statement.